src/pages/filter/ui/gene-region.tsx

```diff
@@ -1,101 +1,120 @@
 import React, { useEffect, useState } from 'react'
```

Comment:
Give it some folder structure:
/filter/ui/panels/function-panel/components/gene-region.tsx
/filter/ui/panels/function-panel/function-panel.tsx
It will be easier to navigate